src/js/helpers.test.js

```diff
@@ -120,6 +121,24 @@ systemctl restart mender-client'`
   });
 });
 
+describe('getDemoDeviceCreationCommand function', () => {
+  const token = `eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJtZW5kZXIudGVuYW50IjoiNWY5YWI0ZWQ4ZjhhMzc0NmYwYTIxNjU1IiwiaXNzIjoiTWVuZGVyIiwic3`;
+  it('should not contain any template string leftovers', () => {
+    let code = getDemoDeviceCreationCommand();
+    expect(code).not.toMatch(/\{([^}]+)\}/);
+    code = getDemoDeviceCreationCommand(token);
+    expect(code).not.toMatch(/\{([^}]+)\}/);
+  });
+  it('should return a sane result', () => {
+    let code = getDemoDeviceCreationCommand();
+    expect(code).toMatch('./demo --client up');
+    code = getDemoDeviceCreationCommand(token);
+    expect(code).toMatch(
+      `TENANT_TOKEN='${token}'\ndocker run -it -p 85:85 -e SERVER_URL='https://localhost' \\\n-e TENANT_TOKEN=$TENANT_TOKEN mendersoftware/mender-client-qemu:latest`
```

Comment:
the port forward `85:85` is present only here, is that on purpose?

Reply:
ah ok I see it in 
```${onboardingReducerState.demoArtifactPort}:${
        onboardingReducerState.demoArtifactPort
      }
```

Reply:
in the tests it is static anyways, so I can expect the `85` 👌